### DISCUSSION TITLE
How can i trigger a webhook manually?

### DISCUSSION THREAD

**Discussion by icaroxavier** [codemagic-ci-cd/codemagic-docs#1204] [Q&A]
The apps have a webhook to integrate via GitHub, BitBucket etc, but i want to trigger via cURL, but i dont know what payload/body i must pass to the link to work, can someone give me a example?
![image](https://user-images.githubusercontent.com/73499961/205133210-d82971c3-1581-4929-8ebe-d0581589ff1e.png)
![image](https://user-images.githubusercontent.com/73499961/205133270-fa7bed8f-3ca8-4733-9a46-125c119b6a0c.png)

**Comment by mikhail-tokarev** [comment]
to trigger build via curl you need to use Rest API - check the documentation https://docs.codemagic.io/rest-api/codemagic-rest-api/
